app/routes/user/Tabs.tsx

```diff
@@ -0,0 +1,55 @@
+import type { FC } from 'react'
+
+type TabsContainerProps = {
+  tabs: {
+    label: string
+    index: number
```

Comment:
Right, but the index can be implicit because you have an array of tabs. You can iterate over them like this:

```mjs
tabs.map((tab, index) => ...)
```

The second (optional) argument to `.map` is the index of the current element.

Reply:
Gotcha, resolving now